packages/familie-header/src/søk/idnrValidator.ts

```diff
@@ -0,0 +1,60 @@
+const elevenDigits = new RegExp('^\\d{11}$')
+
+export const idnr = (digits: string, acceptSynthNr = false) => {
```

Comment:
I think this could have another name, maybe `validerFnr`?

Reply:
Maybe validerIdnr because both Fnr and Dnr are supported